dist/pom.xml

```diff
@@ -29,40 +29,150 @@
-    <dependency>
-       <groupId>com.nvidia</groupId>
-       <artifactId>rapids-4-spark-sql_${scala.binary.version}</artifactId>
-       <version>${project.version}</version>
-    </dependency>
-    <dependency>
-       <groupId>com.nvidia</groupId>
-       <artifactId>rapids-4-spark-shuffle_${scala.binary.version}</artifactId>
-       <version>${project.version}</version>
-    </dependency>
-    <dependency>
-       <groupId>com.nvidia</groupId>
-       <artifactId>rapids-4-spark-shims-aggregator_${scala.binary.version}</artifactId>
-       <version>${project.version}</version>
-    </dependency>
-    <dependency>
-       <groupId>com.nvidia</groupId>
-       <artifactId>rapids-4-spark-udf_${scala.binary.version}</artifactId>
-       <version>${project.version}</version>
-    </dependency>
-    <dependency>
-       <!-- required for conf generation script -->
-       <groupId>org.apache.spark</groupId>
-       <artifactId>spark-sql_${scala.binary.version}</artifactId>
-       <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <!-- required for conf generation script -->
-      <groupId>org.apache.spark</groupId>
-      <artifactId>spark-hive_${scala.binary.version}</artifactId>
-      <scope>provided</scope>
-    </dependency>
-  </dependencies>
+  <profiles>
+    <profile>
+        <id>default</id>
+        <activation>
+            <activeByDefault>true</activeByDefault>
+        </activation>
+        <dependencies>
+            <dependency>
+               <groupId>com.nvidia</groupId>
+               <artifactId>rapids-4-spark-sql_${scala.binary.version}</artifactId>
+               <version>${project.version}</version>
+            </dependency>
+            <dependency>
+               <groupId>com.nvidia</groupId>
+               <artifactId>rapids-4-spark-shuffle_${scala.binary.version}</artifactId>
+               <version>${project.version}</version>
+            </dependency>
+            <dependency>
+               <groupId>com.nvidia</groupId>
+               <artifactId>rapids-4-spark-udf_${scala.binary.version}</artifactId>
+               <version>${project.version}</version>
+            </dependency>
+            <dependency>
+               <groupId>com.nvidia</groupId>
+               <artifactId>rapids-4-spark-shims-aggregator_${scala.binary.version}</artifactId>
+               <version>${project.version}</version>
+            </dependency>
+            <dependency>
+               <!-- required for conf generation script -->
+               <groupId>org.apache.spark</groupId>
+               <artifactId>spark-sql_${scala.binary.version}</artifactId>
+               <scope>provided</scope>
+            </dependency>
+            <dependency>
+              <!-- required for conf generation script -->
+              <groupId>org.apache.spark</groupId>
+              <artifactId>spark-hive_${scala.binary.version}</artifactId>
+              <scope>provided</scope>
+            </dependency>
```

Comment:
Seems like we should be able to move these out of the profiles and into a common `dependencies` section since they're not changing between profiles

Reply:
I made a comment on this in pr description, I agree seems like we should but it wasn't working for some reason. I'm going to investigate more but I've run into that a couple places where it doesn't seem to properly override. In this case it wasn't pulling all the dependencies. I'm going to investigate more but was hoping it wouldn't block this.

Reply:
Yep, definitely not a blocker.  Just surprised this isn't working since we rely on the "profile adds dependencies to base dependencies" behavior elsewhere.